apm-lambda-extension/e2e-testing/e2e_test.go

```diff
@@ -1,11 +1,6 @@
 package e2e_testing
```

Comment:
I restructured the e2e testing code in order to make some utility functions (decompress, unzip, run bash command, ...) available to other packages.